wwwroot/index.html

```diff
@@ -1,257 +1,480 @@
+                </a>
+            </li>
+            <li>
+                <a href="https://discord.gg/nm7YHEZnJs" target="_blank">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="20" height="15.6" viewBox="0 0 20 15.6">
+                        <path id="Path_10" data-name="Path 10" d="M2713.71,192.459a17.682,17.682,0,0,0-4.139-1.349l-.511,1.209a14.721,14.721,0,0,0-2-.177v-.009s-.088,0-.235,0-.235,0-.235,0v.009a14.72,14.72,0,0,0-2,.177l-.511-1.209a17.682,17.682,0,0,0-4.138,1.349,17.043,17.043,0,0,0-3.022,11.625,21.405,21.405,0,0,0,5.02,2.626,14.985,14.985,0,0,0,1.095-1.812,6.016,6.016,0,0,1-1.674-.791l.419-.372a11.416,11.416,0,0,0,5.045,1.139,14.419,14.419,0,0,0,5.041-1.139h0l.419.372a6.02,6.02,0,0,1-1.674.791,14.985,14.985,0,0,0,1.095,1.812,21.4,21.4,0,0,0,5.02-2.626,17.044,17.044,0,0,0-3.022-11.625m-10.2,9.37a2.114,2.114,0,1,1,1.813-2.093,1.968,1.968,0,0,1-1.813,2.093m6.65,0a2.114,2.114,0,1,1,1.814-2.093,1.968,1.968,0,0,1-1.814,2.093" transform="translate(-2696.83 -191.11)" fill="#fff" /> </svg>
+                </a>
+            </li>
+        </ul>
+        <section class="video-section">
+            <div class="video-homepage"> <img class="mobile_video_bg d-block" src="img/video_bg.png" alt="Wasabi Wallet - video"> </div>
+            <div class="container">
+                <div class="row vh-100">
+                    <div class="col-lg-11 offset-lg-1 position-relative">
+                        <div class="floating-home">
+                            <div class="row">
+                                <div class="col-lg-5">
+                                    <h1 class="title"><strong>Wasabi Wallet</strong></h1>
+                                    <p class="mb-4">The most popular Bitcoin wallet for desktop with coinjoin. Free & open-source. Non-custodial.
+                                        <br>Reclaim your privacy today.</p><a href="index.html#download" class="btn btn-primary mt-2 mt-md-0">Download</a> <a href="#bitcoinisnotprivate" class="btn btn-blur affix-arrowright mt-2 mt-md-0">Read more</a> </div>
+                                <div class="col-lg-4 offset-lg-3">
+                                    <div class="box-blur d-none d-lg-block">
+                                        <div class="title-small mb-3">No default coordinator</div>
+                                        <p>Starting from June 1, 2024, Wasabi Wallet will no longer have a default coinjoin coordinator.</p><strong><a href="https://github.com/WalletWasabi" target="_blank" class="affix-arrowright">Don't trust, verify</a></strong> </div>
```

Comment:
this should related to the announcement should link to the blog announcement not the github repo